settings.py

```diff
@@ -185,6 +185,21 @@ def path(*args):
 ET_CLIENT_ID = config('ET_CLIENT_ID', None)
 ET_CLIENT_SECRET = config('ET_CLIENT_SECRET', None)
 
+if EXACTTARGET_USE_SANDBOX:
+    auth_url = 'https://auth-test.exacttargetapis.com/v1/requestToken?legacy=1'
+    wsdl_loc = 'etframework.test.wsdl'
+else:
+    auth_url = 'https://auth.exacttargetapis.com/v1/requestToken?legacy=1'
+    wsdl_loc = 'etframework.wsdl'
+
+SFMC_SETTINGS = {
+    'authenticationurl': auth_url,
+    'wsdl_file_local_loc': str(ROOT_PATH / 'news' / 'backends' / wsdl_loc),
```

Comment:
I agree actually. `pathlib` also has a `joinpaths()` method that is more like `os.path.join`. I'm happy to use that instead.